docs-src/concepts.md

```diff
@@ -112,42 +114,123 @@ TODO
 
 ### GatewayClass
 
-`GatewayClass` is cluster-scoped resource defined by the infrastructure ops.
-This resource represents a category of Gateways that can be instantiated.
+`GatewayClass` ([source code][gatewayclass-src]) is cluster-scoped resource
+defined by the infrastructure ops. This resource represents a class of Gateways
+that can be instantiated. 
+
+[gatewayclass-src]: https://github.com/kubernetes-sigs/service-apis/blob/master/api/v1alpha1/gatewayclass_types.go
+
+> Note: this serves the same function as the `networking.IngressClass` resource.
```

Comment:
Can you link to this resource? I don't see `IngressClass` [here](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.17/) or [here](https://github.com/kubernetes/api/tree/master/networking).

Reply:
Currently in KEP form, theoretically will be part of Ingress V1: https://github.com/kubernetes/enhancements/pull/1457. It would be nice if IngressClass and GatewayClass could closely match, but with IngressClass tied to the Kubernetes release schedule it may be difficult to have a perfect match.

Reply:
Added reference